spec/controllers/projects/classifications_controller_spec.rb

```diff
@@ -49,13 +49,23 @@
     context 'request to be classified can not be found' do
       include_context 'project can be found'
 
-      it'raises a ActiveRecord::RecordNotFound error' do
+      it 'raises a ActiveRecord::RecordNotFound error' do
         expect {
           post :create, params: { project_id: project.id, url_title: 'invalid' }
         }.to raise_error(ActiveRecord::RecordNotFound)
       end
     end
 
+    context 'url_title param not submitted' do
+      include_context 'project can be found'
+
+      it 'raises an ActionController::ParameterMissing error' do
+        expect {
+          post :create, params: { project_id: project.id, info_request_id: '1' }
```

Comment:
This spec is actually useless because we're stubbing `find_by!` – even though the parameter is missing we're not actually calling `url_title` because the stub happens first.